server/etc/httpd/conf.d/pulp_content.conf

```diff
@@ -0,0 +1,25 @@
+
+SSLInsecureRenegotiation on
```

Comment:
In my opinion we should leave this commented out or remove it. Apparently this was added because very old yum clients didn't support secure renegotiation[0] and although the version mentioned in the bug (3.2.25) is newer than RHEL5's version (3.2.22), I believe the Apache configuration we provide should be secure by default. If a user wants to support RHEL5, they can uncomment/add it.

I do also wonder if the RHEL5 version of yum (or rather openssl because I suspect yum uses openssl) has been patched to support secure renegotiation.

[0] https://bugzilla.redhat.com/show_bug.cgi?id=647828